pydantic/main.py

```diff
@@ -593,11 +613,15 @@ def validate_model(  # noqa: C901 (ignore complexity)
                 f'you might need to call {model.__class__.__name__}.update_forward_refs().'
             )
 
-        value = input_data.get(field.alias, _missing)
+        value = _get_input_value(input_data, field.alias)
```

Comment:
I don't really care about the performance of the new ORM functionality since I won't ever be using an ORM (unless I end up in hell or working at a bank).

But I do care a lot about the performance of pydantic. Will this have a performance impact? If so how much and how can we mitigate it.

Reply:
Cool. I'll implement the changes and then we can check if it's OK for Pydantic's standard use.

And later, after this PR, I or anyone else can do more benchmarks for specific ORMs/tools and see if things can be improved there. But we'll have a baseline to start.